packages/nimble-components/src/wafer-map/tests/DataGenerator.ts

```diff
@@ -0,0 +1,56 @@
+import type { WaferMapDie } from '../types';
```

Comment:
Update the file to use nimble's file naming convention `data-generator.ts`

Reply:
applied